modules/email/src/test/java/com/intuit/wasabi/email/EmailModuleTest.java

```diff
@@ -20,9 +20,7 @@
 import com.google.inject.Injector;
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
When I joined the project people always pointed it out, and I think it's nice to have it explicit rather than implicit - importing only specific parts is in general better than general parts. But it's fine, it works.